index.html

```diff
@@ -91,27 +93,29 @@ <h2 class="work-title">Tonic</h2>
             </div>
 
             <!-- 2ND WORK SECTION -->
-            <div class="work-box">
+            <div class="work-box reverse">
```

Comment:
- [ ] It is good that you added this outline in this card. However, the outline(border) that is applied should be the hover effect of all project cards.
![project-card-outline-hover-impl](https://user-images.githubusercontent.com/44057863/167259367-2334c66a-ca32-4866-bd23-a7706cdc9b1c.png)

Reply:
the hover effect has been added to the cards 
thanks